cli/tools/fmt.rs

```diff
@@ -213,9 +255,16 @@ fn files_str(len: usize) -> &'static str {
   }
 }
 
-fn get_config() -> dprint::configuration::Configuration {
-  use dprint::configuration::*;
-  ConfigurationBuilder::new().deno().build()
+fn get_typescript_config(
+) -> dprint_plugin_typescript::configuration::Configuration {
+  dprint_plugin_typescript::configuration::ConfigurationBuilder::new()
+    .deno()
+    .build()
+}
+
+fn get_markdown_config() -> dprint_plugin_markdown::configuration::Configuration
+{
+  dprint_plugin_markdown::configuration::ConfigurationBuilder::new().build()
```

Comment:
dprint-plugin-markdown 0.5.0 has the new configuration to configure the ignore directive text (thanks @bartlomieju for implementing!). You should be able to specify `deno-ignore`, `deno-ignore-start`, and `deno-ignore-end` (see here: https://github.com/dprint/dprint-plugin-markdown/pull/31/files#diff-08092886a134ae07f7991c185445595220f6e624d8d1b3f9c7bf52c7d02e58f0)

Reply:
Awesome, thank you David for quick release :)